wsmaster/che-core-api-workspace-activity/src/main/java/org/eclipse/che/api/workspace/activity/WorkspaceActivityManager.java

```diff
@@ -71,9 +71,9 @@ public WorkspaceActivityManager(
     this.defaultTimeout = timeout;
     if (timeout < MINIMAL_TIMEOUT) {
       LOG.warn(
-          "Default workspace idle timeout has a value of less than "
+          "Value of property \"che.limits.workspace.idle.timeout\" is below recommended minimum ("
               + TimeUnit.MILLISECONDS.toMinutes(MINIMAL_TIMEOUT)
-              + " minutes.");
+              + " minutes). This may cause problems with workspace components startup and can cause premature workspace shutdown.");
```

Comment:
I propose:
`This may cause problems with workspace components startup and/or premature workspace shutdown.`

Reply:
ok